dandelion.json

```
@@ -759,6 +759,10 @@
       "path": "bundles/qr-code-generator/",
       "version": "2.0.0"
     },
+    "representative-company-user": {
+      "path": "bundles/representative-company-user/",
+      "version": "1.0.0.beta.1"
```

According to [Semantic Versioning](https://semver.org/spec/v1.0.0-beta.html), it should be with - or?
```suggestion
      "version": "1.0.0-beta.1"
```

---

jupp, changed it. ty